### ISSUE TITLE
Support for adding web analytics tracker eg: Google analytics

### ISSUE THREAD

**Issue by subintp** [hypertrace/hypertrace-ui#937]
## Use Case
*Currently there is no way to track developer engagement on Hypertrace. Engagement tracking will help us to understand how developers use the apps and also plan the evangelization around the features effectively within the company *

## Proposal
There is should some way to inject this javascript code into hypertrace UI.

**Comment by kotharironak** [comment]
@anandtiwary

**Comment by kotharironak** [comment]
@anandtiwary Can we close this one too?

**Comment by anandtiwary** [comment]
Yes. This is done. I will add a detailed step for enabling/configuring telemetry provider for Hypertrace.

cc @subintp @jyothishjose6190 @jaywalker21
